scripts/ci/Dockerfile.bitbake

```diff
@@ -18,6 +18,7 @@ RUN apt-get update -q && apt-get install -qy \
      libpython-dev \
      libsdl1.2-dev \
      locales \
+     ovmf \
```

Comment:
Cool, are we testing Minnowboard builds now?

Reply:
It's running the full oe-selftests updater tests, so yes. But it's long and fails a lot :) and only on master right now.

I'll merge when the tests I launched today will reach a reasonable point (success or near success).